content/protocols/mqtt.textile

```diff
@@ -0,0 +1,240 @@
+
+To specify the parameter @foo@ with value @bar@ on channel @baz@, the qualified channel name would be @[?foo=bar]baz@. If the channel name already has a qualifier, such as @[meta]log@, then the query string follows the existing qualifier, as in @[meta?foo=bar]log@.
+
+Using this syntax with a non-supported Ably library means that channel parameters are specified for the lifetime of the Channel instance; in order to reference the same channel, but with different channel parameters, it is necessary to get a new Channel instance, using a qualified name that includes the new channel parameters.
+
+Some transports provide raw message payloads - that is, the content of the @data@ attribute of a @Message@ - without the accompanying metadata. That means that the recipient of the message does not have access to the @extras@ or @encoding@ attributes of the message that would ordinarily be used to decode delta message payloads.
+
+In order to assist applications that use these transports, @vcdiff@ decoder libraries can check for the @vcdiff@ header at the start of the message payload as an inexact method of determining whether or not the message is a regular message or a delta. Note that, in order to rely on that check, you need to know that that header will not be present in any valid (uncompressed) message in your app. No valid JSON value, for example, will match the @vcdiff@ header check, so it is safe to perform this sniffing on JSON message payloads.
+
+Read more in the "delta section":/channels/options/delta.
+
+h4(#delta-example). Delta example
+
+```[nodejs]
+  var mqtt = require('mqtt');
+  var { VcdiffDecoder } = require('@ably/vcdiff-decoder');
+
+  var options = {
+    keepalive: 30,
+    username: '{{API_KEY_NAME}}', /* API key's name */
+    password: '{{API_KEY_SECRET}}', /* API key's secret */
+    port: 8883
+  };
+  var client = mqtt.connect('mqtts:mqtt.ably.io', options);
+  var channelName = 'sample-app-mqtt';
+  var channelDecoder = new VcdiffDecoder();
+
+  client.on('message', (_, payload) => {
+    var data = payload;
+
+    try {
+      if (VcdiffDecoder.isDelta(data)) {
+        data = channelDecoder.applyDelta(data).asUint8Array();
+      } else {
+        channelDecoder.setBase(data);
+      }
+    } catch(e) {
+      /* Delta decoder error */
+      console.log(e);
+    }
+
+    /* Process decoded data */
+    console.log(data);
+  });
+
+  client.subscribe(`[?delta=vcdiff]${channelName}`);
+```
+
+h3(#rewind-mqtt). Rewind with MQTT
+
+To use Rewind with MQTT specify the @rewind@ channel option with the value @"1"@ using a querystring parameter, where it will apply to all channels.
```

Comment:
"Rewind" --> "rewind"

This also reads like you can only use rewind with `1` which isn't true. 

For MQTT this also wouldn't apply to all channels (that's the SSE connection string option) - it would only apply to the channels where you qualify the names with a query string.